### PR TITLE
Use lumberjack library to rotate logs and add config that enables it

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]
great changes!

**Review by adambabik — APPROVED** [review]
LGTM!

I would test this PR with iOS and Android before merging, if possible, because we used to have problems with log files permissions.

**Comment by dshulyak** [comment]
Yeah trying to prepare a build for tests. I wonder though what kind of permissions issues we had. As i understand bindings have exactly same permissions to modify file system as a statusgo part. So what was solved by creating geth.log directly in native bindings?

File: params/config.go [hunk 1]
```diff
@@ -378,6 +387,9 @@ func NewNodeConfigWithDefaults(dataDir string, networkID uint64, opts ...Option)
 	c.ListenAddr = ":30303"
 	c.LogEnabled = true
 	c.LogLevel = "INFO"
+	c.LogMaxSize = 100
```
Comment (mandrigin) [inline params/config.go:390]: is it megabytes, right?

**Comment by mandrigin** [comment]
@dshulyak just do double-check, if I won't specify the log size limit when calling `StartNode`, but will specify other params, it will be limited to 100MB, right?

File: params/config.go (hunk 1 above)
Comment (dshulyak) [reply params/config.go:390]: yes

Comment (mandrigin) [reply params/config.go:390]: thanks!

**Comment by dshulyak** [comment]
> @dshulyak just do double-check, if I won't specify the log size limit when calling `StartNode`, but will specify other params, it will be limited to 100MB, right?

yes, by default 100mb, at most 2 last rotated logs will be kept and rotated logs will be compressed

**Pull request merged**